File: Go/015_Program/main.go
package main

import "fmt"

func main() {
    fmt.Println("=== Go Program 015 ===")
    fmt.Println("CLI utility program")

    var input string
    fmt.Print("Enter command: ")
    fmt.Scanln(&input)

    fmt.Printf("Processing: %s\n", input)
}

File: Rust/015_Program/main.rs
fn main() {
    println!("=== Rust Program 015 ===");
    println!("Basic Rust utility");

    let mut input = String::new();
    println!("Enter input:");

    std::io::stdin().read_line(&mut input).unwrap();
    println!("You entered: {}", input.trim());
}
